nautobot/core/graphql/__init__.py

```diff
@@ -46,7 +46,7 @@ def execute_saved_query(saved_query_slug, **kwargs):
     Returns:
         GraphQL Object: Result for query
     """
-    query = GraphQLQuery.objects.get(slug=saved_query_slug)
+    query = GraphQLQuery.objects.get(name=saved_query_slug)
```

Comment:
Presumably `saved_query_slug` should be changed to `saved_query_name`? Also this API is documented in `nautobot/docs/plugins/development.md` and that probably needs to be updated to call this out as an API change.